_docs/quickstart.md

```diff
@@ -260,88 +264,52 @@ $ flux exec -r 3 flux getattr rank
 3
 ```
 
-To view processes launched using `flux exec`, a `flux ps` program is
-provided
-
-```console
-$ flux exec sleep 100 &
-[1] 161298
-$ $ flux ps
-OWNER     RANK       PID  COMMAND
-none         0    111760  /bin/bash
-8E20D        0    162394  sleep
-8E20D        3    162395  sleep
-8E20D        2    162396  sleep
-8E20D        1    162397  sleep
-```
-
-The `OWNER` field refers to a UUID of the requesting entity in the system
-not  a user identity.
-
-
-The second method for launching parallel jobs is a prototype based on
-use of the Flux KVS for parallel job management termed "WRECK". The
-wreck prototype consists of a `flux wreckrun` frontend command,, and
-a `flux wreck` utility for operating and querying jobs run under the
+The second method for launching and submitting jobs is a Minimal Job Submission Tool named "mini". The "mini" prototype consists of a `flux mini` frontend command, and a `flux job` utility for operating and querying jobs run under the
```

Comment:
Minor correction: I don't think `flux-mini` is a prototype, but it is a "minimal" tool as you say. Also `flux-job` isn't part of `flux-mini` but is a low-level tool for querying job information.